apps/src/templates/CensusForm.jsx

```diff
@@ -0,0 +1,158 @@
+import React, {Component} from 'react';
+import Checkbox from './forms/Checkbox';
+import Button from './Button';
+import color from "../util/color";
+import i18n from "@cdo/locale";
+
+const styles = {
+  question: {
+    fontSize: 16,
+    fontFamily: '"Gotham 4r", sans-serif',
+    color: color.charcoal,
+    paddingBottom: 10,
+    paddingTop: 10
+  },
+  definition:{
+    fontSize: 14,
+    fontFamily: '"Gotham 4r", sans-serif',
+    color: color.charcoal,
+  },
+  pledge: {
+    fontSize: 18,
+    fontFamily: '"Gotham 7r", sans-serif',
+    color: color.charcoal,
+    paddingBottom: 10,
+    paddingTop: 10
+  },
+  option: {
+    fontFamily: '"Gotham 4r", sans-serif',
+    color: color.charcoal,
+  },
+  checkbox: {
+    width: 25,
+    height: 25,
+    marginTop: -2,
+    paddingRight: 5
+  },
+};
+
+class CensusForm extends Component{
+
+  constructor(props) {
+    super(props);
+    this.state = {value: ''};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
```

Comment:
You can avoid needing a constructor by having the following:
```js
state = {
    value: ''
};

handleChange = event => {
  this.setState({value: event.target.value});
}

Reply:
This is a pattern we've been using more recently. One example of it: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/teacherDashboard/EditSectionForm.jsx#L57

Reply:
I see you also used this pattern elsewhere in Checkbox.

Also just noticed another nit: there should be a space after Component on line 39